ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTest.java

```diff
@@ -464,6 +457,9 @@ public void ethGetUncleCountByBlockNumberPending() throws Exception {
 
   @Test
   public void ethGetUncleCountByBlockNumberPendingNoData() throws Exception {
+    when(blockchainQueries.headBlockNumber()).thenReturn(0L);
```

Comment:
The test had a hidden dependency on another test. It relied on one of the tests before it to set up the mocking. After the change to non-static variables this dependency became visible and the tests was failing. It was necessary to set up its own mocking configuration.